Tests/SwiftFormatPrettyPrintTests/BinaryOperatorExprTests.swift

```diff
@@ -16,8 +16,9 @@ public class BinaryOperatorExprTests: PrettyPrintTestCase {
 
     let expected10 =
       """
-      x = 1 + 8
-        - 9
+      x =
+        1 + 8
+          - 9
         ^*^ 5
```

Comment:
Shouldn't these last two lines now also line up with the `- 9`?

Reply:
Yeah, that's what I would expect. This is a result of the way our sequence folding works though. The "^*^" operator is unknown, so it's precedence is unknown (represented by `nil`) when this sequence expr is folded. When the unrecognized operator is encountered, the sequence containing the `=` is split there.

I would expect the sequence tree for this expr to look something like...
```
    =
  /   \
x     [RHS]  
```

But instead, it looks like...
```
        "\"
        /  \
       *   10
      / \
   "^*^" 4
   /  \
  =    5
 / \
x  [1 + 8 - 9]
```

Because of the shape of that tree of sequence exprs, the "^*^" operator and everything following it isn't included in the `break(open(.continuation))` on the RHS of the assignment operator. I think solving this would require folding the sequence such that everything to the right of the "=" is included in the right subtree sequence expr so that it can be included in that open break.

Reply:
Riiight, got it. Ok, I'm somewhat less worried about it now. I may look at having unknown operators fold just below `AssignmentPrecedence` (or `TernaryPrecedence`) instead of where they do now—this doesn't risk divergence from the compiler because an unknown operator is an error there, but it would make the formatting results look better.